skein/src/lib.rs

```diff
@@ -0,0 +1,260 @@
+#![no_std]
```

Comment:
Add a basic crate documentation, ideally, with a simple usage example. Also it would be nice to add the following crate attributes:
```rust
#![doc(
    html_logo_url = "https://raw.githubusercontent.com/RustCrypto/media/6ee8e381/logo.svg",
    html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/media/6ee8e381/logo.svg"
)]
#![warn(missing_docs, rust_2018_idioms)]
```